implementations/rust/ockam/ockam_api/src/control_api/backend/ticket.rs

```diff
@@ -197,30 +219,35 @@ async fn create_encoded_ticket(
 
 #[utoipa::path(
     post,
-    operation_id = "enroll_ticket",
-    summary = "Enroll a Ticket",
-    path = "/{node}/ticket",
-    tags = ["ticket"],
+    operation_id = "project_enroll",
+    summary = "Enroll to a Project using a Ticket",
+    description =
+"This API enroll a node to a Project using the provided Ticket.
+Note that this API imports the Project in the node database, but the node won't be able to use
+it until it restarts.
+The easiest way to use a ticket is to specify the ticket directly during the node creation.",
```

Comment:
Should we specify this if the user of the API has no way to create a node?

Reply:
A user could create the ticket via the API and then spawn a node

Reply:
My question is "how can they create a node?". The API does not offer this possibility, right?

Reply:
No, the API doesn't offer this possibility, and most likely it won't be its responsibility 🤔

Reply:
I would remove this sentence from the API doc, because for a user of the API, this is outside of their realm.